qa/tasks/vstart_runner.py

```diff
@@ -530,8 +530,21 @@ def safe_kill(pid):
         else:
             raise
 
+def using_localhost(config_path="./ceph.conf"):
+    """
+    If the ceph cluster is using the localhost IP as mon host, will must disable ns unsharing
+    """
+    with open(config_path) as f:
+        for line in f:
+            asok_conf = re.search("^\s*mon\s+host\s*=\s*\[(.*:127\.0\.0\.1:.*)+\]", line)
```

Comment:
```suggestion
            asok_conf = re.match(r'^\s*mon host\s*=\s*\[((v1|v2):127\.0\.0\.1:\d+,?)+\]', line)
```

for 5 reasons:

1. what you are doing is to "match" instead of "search"ing a pattern
2. i think the config key will not have more than one spaces in it. as we will replace all spaces with "_" when parsing a conf file.
3. to be explicit, as only "v1" and "v2" are allowed before an "address:port".
4. port is composed with numbers
5. delimiter between mon host addresses is ","

Reply:
Yeah, make sense and will fix it. Thanks.